--- ./vendor/resistance/lib/resistance.js	2012-12-25 08:22:44.000000000 -0800
+++ ./build/vendor/resistance.js	2012-12-25 08:28:58.000000000 -0800
@@ -1,3 +1,6 @@
+$.fn.textext.resistance = (function()
+{
+
 var instant = function(fn) {
   setTimeout(fn, 0);
 };
@@ -7,11 +10,11 @@
   var completed = 0;
   var data = [];
   var iterate = function() {
-    fns[completed](function(results) {
+    fns[completed](function(err, results) {
       data[completed] = results;
-      if (++completed == fns.length) {
+      if (err || ++completed == fns.length) {
         // this is preferred for .apply but for size, we can use data
-        if (callback) callback.apply(data, data);
+        if (callback) callback.apply(data, [ err ].concat(data));
       } else {
         iterate();
       }
@@ -27,10 +30,10 @@
   var data = [];
   var iterate = function() {
     fns[started]((function(i) {
-      return function(results) {
+      return function(err, results) {
         data[i] = results;
-        if (++completed == fns.length) {
-          if (callback) callback.apply(data, data);
+        if (err || ++completed == fns.length) {
+          if (callback) callback.apply(data, [ err ].concat(data));
           return;
         }
       };
@@ -67,8 +70,12 @@
   };
 };
 
-var R = {
+return {
   series: runSeries,
   parallel: runParallel,
   queue: queue
-};
+}
+
+
+})();
+
